### PR TITLE
Gabriel Shapiro - Javascript 1 Project 

### PR DESCRIPTION
**Pull request by lex-node**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```
Comment (jacobAngulo) [inline assignments/arrays.js:66]: I think the idea here was to have you do a for loop through it to find the car with the matching id but this is technically correct.

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`${inventory[lastCar].car_make} ${inventory[lastCar].car_model}`);
```
Comment (jacobAngulo) [inline assignments/arrays.js]: this is actually returning the first car in the array when you want the last one

Comment (jacobAngulo) [reply assignments/arrays.js]: there's a way to grab the last one by using the length of the array

File: assignments/arrays.js [hunk 3]
```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 6 ====
-// A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
+// A buyer is interested in seeing only BMW and Audi cars within the inventory.
+// Return an array that only contains BMW and Audi cars.
+// Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi = [];
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].car_make === "BMw" || inventory[i].car_make === "Audi") {
```
Comment (jacobAngulo) [inline assignments/arrays.js]: I'm only seeing audis in the console, too bad there's no spellcheck in webStorm

Comment (jacobAngulo) [reply assignments/arrays.js]: I kid I kid

Comment (lex-node) [reply assignments/arrays.js]: Dang it. Had a lowercase m. Fixed in latest commit.

**Review by jacobAngulo — APPROVED** [review]
MVP and Stretch woo! everything looks good and clean, impressed with the whole objects parts. keep it up!

File: assignments/arrays.js (hunk 3 above)
Comment (jacobAngulo) [reply assignments/arrays.js]: 🔥

**Pull request merged**